salsah/src/public/js/jquery.htmleditor.js

```diff
@@ -165,7 +165,8 @@
 							
 						}
 					},
-					toolbar: [ ['Source', 'Bold', 'Italic', 'Underline', 'Strike', 'Subscript', 'Superscript', '-','RemoveFormat', 'Link', 'Unlink'] ] // configuration for toolbar buttons, must conform to `STANDARD_MAPPING`
+					toolbar: [ ['Source', 'Bold', 'Italic', 'Underline', 'Strike', 'Subscript', 'Superscript', '-','RemoveFormat', 'Link', 'Unlink'] ], // configuration for toolbar buttons, must conform to `STANDARD_MAPPING`
+					removeButtons: ''
```

Comment:
concerning the log that shows up ("TypeError: can't access dead object" `htmlPanel.js(1082,17)`) .

As we don't load that library directly (no `<script src="htmlPanel.js">`), do you know where it is loaded from?

From this [MDN page](https://developer.mozilla.org/ca/docs/Web/JavaScript/Reference/Errors/Dead_object), it looks like it comes from firefox add-ons.

I don't get this error, but maybe running the same on firefox with add-ons disabled or on another firefox-based browser (like the [developer edition](https://www.mozilla.org/en-US/firefox/developer/)) would help to decide if this message is linked to our code or not.